### PR TITLE
New fcu 

### PR DESCRIPTION
**Pull request by kellyhw**
@Ciaran-Oflannagain @kevin-hereworks @cstirdivant

### CODE REVIEW

**Comment by cstirdivant** [comment]
@shambergoldstein @ccquigley

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 1]
```diff
@@ -1733,6 +1733,23 @@ FCU_DFSS_RHDHC_HT2RC_DX2RC:
...
+  description: "Fan coil unit with zone temperature control, CO2 and humidity monitoring, heating water and chilled water valve monitoring."
+  is_canonical: true
+  implements:
+  - FCU
+  - DFVSC
+  - RTM
+  - CO2M
+  - ZTC
+  - CHWZTC
+  - CHWSDC
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1747]: how is the chilled water valve controlling to both the zone temp and the supply temp?

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 2]
```diff
@@ -1733,6 +1733,23 @@ FCU_DFSS_RHDHC_HT2RC_DX2RC:
...
+  - VSC
+  - SHWTC
+  - SCHWTC
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1750]: These water points are typicallly used on things like chillers. The FCU itself is not supplying water to anything. Please clarify what you are trying to represent here.

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -1408,6 +1408,7 @@ SCHWTC:
   - cooling_percentage_command
   - cooling_request_count
   - cooling_run_command
+  - cooling_run_status
```
Comment (shambergoldstein) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1411]: What is the analytical benefit in bringing in this field? You should be able to use the chiller run statuses to infer if cooling is running.

Comment (kellyhw) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1411]: @shambergoldstein thank you for your review on that, cooling_run_status for SCHWTC request will be deleted/closed.

Comment (shambergoldstein) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1411]: please remove it from this PR as well, then I should be good to approve the FCU type

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**